frontend/src/pages/list/SalesReport.tsx

```diff
@@ -128,15 +152,13 @@ export default function SalesReport() {
     []
   );
   const [dates, setDates] = useState<any>(null);
-  const [isConfirmationPopupVisible, setIsConfirmationPopupVisible] =
-    useState<boolean>(false);
 
   const onAPIResponse = (response: GetSalesReportResp) => {
     const salesReport = APIToInternalSalesReportConversion(response);
-    console.log(salesReport);
+    //console.log(salesReport);
     setTotalsData([salesReport.totalRow]);
     setDailyData(salesReport.dailySummaryRows);
-    setTopBooksData(salesReport.topBooksRows);
+    setTopBooksData(salesReport.topBooksRows.slice(0, 10));
```

Comment:
We shouldn't have to do this, cause the backend only sends 10 books, right? I also want to make sure there aren't issues when there are fewer than 10 books

Reply:
Yea we shouldn't have to do this, but I felt like a second safegaurd doesn't hurt right. Also it seems like if there is an array of length 0 or less than 10 it doesn't care. If its a length more than 10 than it does.